tasks/shared.py

```diff
@@ -180,6 +186,15 @@ def check_environment(context: Context) -> dict:
     return params
 
 
+def get_compose_cmd(namespace: Namespace) -> str:
+    profile = ""
+    if namespace == Namespace.DEV:
+        profile = "--profile dev"
+    if profile:
+        return f"docker compose {profile}"
+    return "docker compose"
```

Comment:
An alternative here would be to add this command as a property to the `Namespace` enum it could get rid of some of the extra lines throughout.

So instead of
```python
def load_infrastructure_schema(
    context: Context, database: str, namespace: Namespace, add_wait: bool = True, target: str = "models/base"
) -> None:
    with context.cd(ESCAPED_REPO_PATH):
        compose_files_cmd = build_compose_files_cmd(database=database, namespace=namespace)
        compose_cmd = get_compose_cmd(namespace=namespace)
        base_cmd = f"{get_env_vars(context, namespace=namespace)} {compose_cmd} {compose_files_cmd} -p {BUILD_NAME}"
```

We'd have:

```python
def load_infrastructure_schema(
    context: Context, database: str, namespace: Namespace, add_wait: bool = True, target: str = "models/base"
) -> None:
    with context.cd(ESCAPED_REPO_PATH):
        compose_files_cmd = build_compose_files_cmd(database=database, namespace=namespace)
        base_cmd = f"{get_env_vars(context, namespace=namespace)} {namespace.compose_cmd} {compose_files_cmd} -p {BUILD_NAME}"
```

Reply:
I think the function `get_compose_cmd` could help with more than just the profile in the future so for now i would prefer to keep it as it